tests/pkg_tweetnacl/Makefile

```diff
@@ -0,0 +1,22 @@
+# name of your application
+APPLICATION = tweetnacl_test
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP '-DTHREAD_STACKSIZE_MAIN=(THREAD_STACKSIZE_DEFAULT + 2048)'
```

Comment:
Increased stack size is required for tweetnacl? Is this documented somewhere? I would add at least a comment here.